Makefile

```diff
@@ -26,11 +26,25 @@ init:
 
 linter:
 	@echo ──────────── Run linter ───────────────────────
-	@cargo +nightly clippy --all-targets -- --no-deps -D warnings -A "clippy::missing_safety_doc"
+	@cargo +nightly clippy --all-targets -- --no-deps -D warnings -A "clippy::missing_safety_doc" -D warnings -A "clippy::too_many_arguments"
 
 pre-commit: fmt linter test
 
 test: build
-	@if [ ! -f "./target/fungible_token-0.1.0.wasm" ]; then wget "https://github.com/gear-dapps/fungible-token/releases/download/0.1.0/fungible_token-0.1.0.wasm" -O "./target/fungible_token-0.1.0.wasm"; fi
+	@if [ ! -f "./target/ft_main.wasm" ]; then\
```

Comment:
This check forces us to not redownload a binary after`ft_main-*.opt.wasm` changes its version.